src/js/views.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var $ = window.$;
+var L = window.L;
+var _ = window._;
+var Backbone = window.Backbone;
+
+window.MapView = Backbone.View.extend({
+    initialize: function(options) {
+        var map = L.map('map', {
+            center: options.model.get('initialMapCenter'),
```

Comment:
Here's another spot where you should be able to use `this.model` instead of having to pass in and access `options`.